### ISSUE TITLE
Add aria-labels to Buttons

### ISSUE THREAD

**Issue by tbollen** [tbollen/arcane-atlas#45]
The button lacks an accessible label. While it has visible text "Remove Widget", consider adding an `aria-label` or `title` attribute that includes the widget ID for screen reader users to better understand which widget will be removed:

```svelte
<Button
	variant="destructive"
	aria-label="Remove missing widget {widget.name}"
	onclick={() => {
		removeItem(widget.id);
	}}><Icon icon="mdi:delete" /> Remove Widget</Button
>
```

_Originally posted by @Copilot in https://github.com/tbollen/arcane-atlas/pull/44#discussion_r2572915804_
